rotkehlchen/globaldb/cache.py

```diff
@@ -1,75 +1,92 @@
 """Functions dealing with the general_cache table of the Global DB"""
+import sqlite3
 from collections.abc import Iterable
 from typing import Optional, Union
+
 from rotkehlchen.chain.evm.constants import ZERO_ADDRESS
 from rotkehlchen.chain.evm.types import string_to_evm_address
-
 from rotkehlchen.db.drivers.gevent import DBCursor
+from rotkehlchen.errors.misc import InputError
 from rotkehlchen.types import ChecksumEvmAddress, GeneralCacheType, Timestamp
 from rotkehlchen.utils.misc import ts_now
 
+UNIQUE_CACHE = 'unique_cache'
+GENERAL_CACHE = 'general_cache'
+UNIQUE_CACHE_KEYS = ['TEST_KEY']
+''' UNIQUE_CACHE_KEYS: set[GeneralCacheType] = {GeneralCacheType.CURVE_POOL_ADDRESS,
+    GeneralCacheType.MAKERDAO_VAULT_ILK,GeneralCacheType.CURVE_GAUGE_ADDRESS}'''
```

Comment:
You didn't change here. We don't leave commented code and it doesn't make much sense to have `TEST_KEY` here